### PR TITLE
Check for conflicts in parent commands

### PR DESCRIPTION
**Pull request by shadowspawn**
# Pull Request

## Problem

Only testing for conflicts in leaf command.

See: #1710

## Solution

Follow same pattern as checking for required options:
- run up parents checking them as well
- call in dispatch to external command too

## ChangeLog

Fixes:
- detect conflicts in parent commands of called subcommand (#1710)

### CODE REVIEW

File: lib/command.js [hunk 1]
```diff
@@ -1340,6 +1341,19 @@ Expecting one of '${allowedValues.join("', '")}'`);
     });
   }
 
+  /**
+   * Display an error message if conflicting options are used together.
+   * Called after checking for help flags in leaf subcommand.
+   *
+   * @api private
+   */
+  _checkForConflictingOptions(cmd) {
```
Comment (abetomo) [inline lib/command.js:1350]: I don't think the `cmd` argument is necessary.

Comment (shadowspawn) [reply lib/command.js:1350]: Oops, left over from an earlier version of code. Thanks. I should have reread it!

Comment (shadowspawn) [reply lib/command.js]: Fixed.

**Review by abetomo — APPROVED** [review]
LGTM!

**Pull request merged**